src/main/java/com/beerjournal/breweriana/item/rating/RatingController.java

```diff
@@ -0,0 +1,37 @@
+package com.beerjournal.breweriana.item.rating;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Collection;
+
+@RestController
+@RequestMapping("/api/ratings")
+@RequiredArgsConstructor
+public class RatingController {
+
+    private final RatingService ratingService;
+
+    @GetMapping("item/{itemId}")
```

Comment:
bad order in rest api, I think it should be `/api/item/{id}/ratings` or `/api/ratings?itemId`. But I would choose the first one